### PR TITLE
Update to operator-sdk 0.15

### PR DESCRIPTION
**Pull request by alexeykazakov**
Updates operator-sdk to 0.15.2

- https://github.com/codeready-toolchain/host-operator/pull/150
- https://github.com/codeready-toolchain/member-operator/pull/128
- https://github.com/codeready-toolchain/toolchain-operator/pull/68

### CODE REVIEW

**Comment by MatousJobanek** [comment]
I pushed a new commit to your branch: 180a6f5 
Now, it should work for all cases:
* when api types are stored in api repo (host and member)
* when api types are stored in operator repo (toolchain-operator)
* when the script is called using `curl` command
* when the script is called locally
* for generating base template `0.0.1`
* for generating nightly bundles as well as new release manifests

I also pushed the result of the regenerated changes in host-operator PR: https://github.com/codeready-toolchain/host-operator/pull/150/commits/66776f40c9d469dd3893dcd96481c558ad6775bf

Next things to do:
1. Modify toolchain-operator to be able to work with the new version of `operator-sdk` (add necessary annotations into the types)
2. merge all PRs
3. create PRs changing version in `openshift-ci/Dockerfile.tools`
4. merge those too
5. hope that everything will work :-) 

I'll continue working on the first step it on Monday ;-)

**Comment by MatousJobanek** [comment]
The first step is done: https://github.com/codeready-toolchain/toolchain-operator/pull/68
I also modified the README so it reflects the latest changes: 41f3960

**Comment by xcoulon** [comment]
thanks for the fixes, @MatousJobanek

File: scripts/README.adoc [hunk 1]
```diff
@@ -22,11 +22,23 @@ NOTE: This script requires multiple parameters - see help for more information.
...
+* The operator bundle that is used as the base template for all next releases.
 * Some "hack" files, including:
 ** `deploy_csv.yaml` that helps in deploying the current version of the operator bundle to the cluster. It's the same file as is generated using <<generate-deploy-hack,generate-deploy-hack.sh>> script.
 ** `install_operator.yaml` that installs the operator using `OperatorGroup` and `Subscription` resources.
 
+Under the hood, it runs the `operator-sdk` command with specific parameters.
+Based on the fact if the operator repository contains CRD types (if it contains `pkg/apis/toolchain/v1alpha1` directory) it decides where the `operator-sdk` command will be executed.
+
+* If the repository contains the CRD types (in case of toolchain-operator) then the command is executed inside of the operator repo.
+* If the repository doesn't contain the CRD types (in case of hosted toolchain operators) then the command is executed inside of codeready-toolchain/api repo. To do that, the script:
```
Comment (tinakurian) [inline scripts/README.adoc]:
```suggestion
* If the repository doesn't contain the CRD types (e.g., hosted toolchain operators) then the command is executed inside of the codeready-toolchain/api repo. To do that, the script:
```

File: scripts/README.adoc [hunk 2]
```diff
@@ -22,11 +22,23 @@ NOTE: This script requires multiple parameters - see help for more information.
...
 It generates:
 
-* The operator bundle that is used as the base template for all next releases. Under the hood, it runs the `operator-sdk` command with specific parameters.
+* The operator bundle that is used as the base template for all next releases.
 * Some "hack" files, including:
 ** `deploy_csv.yaml` that helps in deploying the current version of the operator bundle to the cluster. It's the same file as is generated using <<generate-deploy-hack,generate-deploy-hack.sh>> script.
 ** `install_operator.yaml` that installs the operator using `OperatorGroup` and `Subscription` resources.
 
+Under the hood, it runs the `operator-sdk` command with specific parameters.
+Based on the fact if the operator repository contains CRD types (if it contains `pkg/apis/toolchain/v1alpha1` directory) it decides where the `operator-sdk` command will be executed.
```
Comment (tinakurian) [inline scripts/README.adoc]:
```suggestion
If the operator repository contains CRD types (i.e., contains the `pkg/apis/toolchain/v1alpha1` directory) it decides where the `operator-sdk` command will be executed.
```

File: scripts/README.adoc [hunk 3]
```diff
@@ -22,11 +22,23 @@ NOTE: This script requires multiple parameters - see help for more information.
...
+
+* If the repository contains the CRD types (in case of toolchain-operator) then the command is executed inside of the operator repo.
```
Comment (tinakurian) [inline scripts/README.adoc]:
```suggestion
* If the repository contains the CRD types (e.g., toolchain-operator) then the command is executed inside of the operator repo.
```

File: scripts/README.adoc (hunk 1 above)
Comment (MatousJobanek) [reply scripts/README.adoc]: it's not an example - it's really the only case

File: scripts/README.adoc (hunk 3 above)
Comment (MatousJobanek) [reply scripts/README.adoc]: same as the previous - it's not an example, but the specific case

Comment (tinakurian) [reply scripts/README.adoc]: @MatousJobanek in that case e.g., can be `i.e.,`

File: scripts/README.adoc (hunk 2 above)
Comment (tinakurian) [reply scripts/README.adoc]: @MatousJobanek in that case e.g., can be i.e.,

File: scripts/README.adoc (hunk 1 above)
Comment (tinakurian) [reply scripts/README.adoc]: @MatousJobanek in that case e.g., can be i.e.,

Comment (MatousJobanek) [reply scripts/README.adoc]: @tinakurian could you please propose the change as a suggestion?

**Pull request merged**